src/Axe.Windows.Actions/Contexts/ElementContext.cs

```diff
@@ -89,7 +88,7 @@ protected virtual void Dispose(bool disposing)
                     this.ProcessName = null;
                     if (this.DataContext != null)
                     {
-                        this.DataContext?.Dispose();
+                        this.DataContext = null;
```

Comment:
>null [](start = 43, length = 4)

The setter already calls Dispose, so there's no functional change here (other than setting DataContext to null)